spec/features/bulk_updates/bulk_updates_science_keywords_spec.rb

```diff
@@ -390,11 +390,11 @@
         end
 
         it 'displays the updated keywords' do
-          within '.science-keywords-preview' do
+          within '#metadata-preview' do
             expect(page).to have_no_content('EARTH SCIENCE ATMOSPHERIC TEMPERATURE SURFACE TEMPERATURE', normalize_ws: true)
             expect(page).to have_no_content('EARTH SCIENCE SOLID EARTH ROCKS/MINERALS/CRYSTALS SEDIMENTARY ROCKS', normalize_ws: true)
 
-            expect(page).to have_content('EARTH SCIENCE ATMOSPHERE AEROSOLS', normalize_ws: true)
+            expect(page).to have_no_content('EARTH SCIENCE ATMOSPHERE AEROSOLS', normalize_ws: true)
```

Comment:
Line 394-397 should be split up to test for each keyword, i,e. EARTH SCIENCE, ATMOSPHERIC TEMPERATURE , SURFACE TEMPERATURE, etc.